.github/workflows/ci.yaml

```diff
@@ -32,56 +32,61 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       tags: ${{ steps.meta.outputs.tags }}
+
     steps:
     - name: Checkout
       uses: actions/checkout@v2
 
-    # Generates tags https://github.com/crazy-max/ghaction-docker-meta
-    - name: Docker meta
+    - name: Generate Docker Tags
       id: meta
-      uses: crazy-max/ghaction-docker-meta@v3.1.0
+      uses: docker/metadata-action@v3
```

Comment:
`crazy-max` sounds cool but I'm glad we're back to using a formally-supported org 😂\